Source/Particles/Pusher/ImplicitPushPX.cpp

```diff
@@ -150,6 +151,8 @@ namespace {
         zp = zp_n + dzp;
         setPosition(ip, xp, yp, zp);
 
```

Comment:
Add a comment

```suggestion
// If a suborbit starts out of bounds, propagate it ballistically (to avoid field gathers in the guard cells)
// and so it cannot be turned back into the domain.
```